README.md

```diff
@@ -12,73 +14,70 @@
+1. A libp2p **DA network** with [Bridge Nodes](#bridge-nodes) which process blocks from the underlying p2p consensus network and [Light Nodes](#celestia-light-nodes), which do data availability sampling on Bridge Nodes' blocks.
+2. A p2p **Consensus network** with [**Validators Nodes**](#bridge-validator-nodes) (which are part of Bridge Nodes), that handles the underlying consensus and block production. 
 
-![Network Overview](diagrams/NetworkOverview.png)
+> Note that mainnet may look very different from this devnet implementation, as the architecture continues to be improved. You can read more about devnet decisions here ([see ADR](https://github.com/celestiaorg/celestia-node/blob/main/docs/adr/adr-003-march2022-testnet.md)).
 
-> It’s important to note that mainnet may look very different from this devnet implementation, as the architecture continues to be improved. You can read more about devnet decisions here and here (link to ADR). 
+### Celestia (Validator) Bridge Nodes
+![Bridge Nodes](diagrams/BridgeNodes.png)
+Specifically, Bridge Nodes: 
 
-## Celestia Light Nodes
+1. Import and process “raw” headers & blocks from a trusted Core process in the Consensus network. Bridge Nodes can run this Core process internally or simply connect to a remote endpoint. Bridge Nodes also have the option of being an active validator in the Consensus network.
+2. Validate and erasure code the "raw" blocks
+3. Produce block shares with DA metadata to Light Nodes in the DA network.
 
-Light nodes (CLN) ensure data availability and can publish transactions. This is the most ubiquitous way to interact with the Celestia network.
+From an implementation perspective, Bridge Nodes run two separate processes:
 
-Specifically, Light Nodes: 
+1. Celestia App with Celestia Core ([see repo](https://github.com/celestiaorg/celestia-app))
+    - **Celestia App** is the state machine where the application and the proof-of-stake logic is run. Celestia App is built on [Cosmos SDK](https://docs.cosmos.network/) and also encompasses **Celestia Core**.
+    - **Celestia Core** is the state interaction, consensus and block production layer. Celestia Core is built on [Tendermint Core](https://docs.tendermint.com/), modified to store (1) invalid transactions and (2) data roots of erasure coded blocks, among other changes ([see ADRs](https://github.com/celestiaorg/celestia-core/tree/master/docs/celestia-architecture)).
+2. Celestia Node ([see repo](https://github.com/celestiaorg/celestia-node))
+    - **Celestia Node** augments the above with a separate libp2p network that serves data availability sampling requests. The team sometimes refer to this as the "halo" network.
 
-1. Connect to a `Celestia Bridge Node` in the DA network. *Note: Light Nodes do not communicate with each other, but only with Bridge Nodes.*
+### Celestia Light Nodes
+![Light Nodes](diagrams/LightNodes.png)
+
+Specifically, Light Nodes:
```

Comment:
```suggestion
In the Devnet, Light Nodes:
```

I only want to specify that this is devnet architecture and will change (specifically the way that (1) happens.